website/docs/components/table/advanced-table/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,243 @@
+
+![End alignment example within a table with a date and more options](/assets/components/table/advanced-table/end-alignment-example-02.png)
+
+!!! Dont
+
+Don’t end align content that is variable in length. This can make the content more difficult to read by forcing an unnatural [reading pattern](/patterns/button-organization?tab=research#layout-and-reading-patterns).
+
+![End alignment with content that is variable in length](/assets/components/table/advanced-table/end-alignment-variable-length.png)
+!!!
+
+### Other alignment methods
+
+We don’t recommend center or justified alignment of content within a cell or advanced table. These alignment methods can result in the content being difficult to read, especially if it is variable in length.
+
+!!! Dont
+
+Don’t center header labels or cell content within a table.
+
+![Example of centered content within an Advanced Table](/assets/components/table/center-justified-alignment.png)
+!!!
+
+## Expandable rows
+
+### Striping 
+
+![Table striping examples](/assets/components/table/advanced-table/advanced-table-striping.png)
+
+Striping enhances readability by alternating row colors, making it easier to scan tabular data.
+- Non-Nested Advanced Tables: Striping starts with the second row, distinguishing it from the header.
+- Nested Advanced Tables:Child rows are automatically striped, while parent rows remain unstriped to visually reinforce hierarchy. This behavior cannot be disabled.
+
+Striping improves clarity for numerical or text-heavy data and ensures hierarchical relationships are visually distinct. For non-nested Advanced Tables, striping is optional and must be enabled with an argument. Always verify adequate color contrast when using striping.
+
+!!! Info
+
+Ensure that nested components within striped rows continue to maintain adequate color contrast with the striped background.
+!!!
+
+### Expandable rows
+
+Expandable rows allow users to expand or collapse additional content within the table structure.
+
+![Advanced Table expandable rows examples](/assets/components/table/advanced-table/expandable-rows.png) 
+
+### When to use
+
+- When organizing hierarchical data into parent-child relationships, allowing users to expand or collapse details as needed.
+
+!!! Dont
+
+Avoid when data does not require structured parent-child relationships.
+
+![Advanced Table column placement example](/assets/components/table/advanced-table/advanced-table-dont-parent-nested.png)
+!!!
+
+## Interaction
+
+- Expanding a parent row reveals its child rows, maintaining indentation for clear relationships.
+- Users can navigate between cells using arrow keys. To expand or collapse a row, they must first focus on the expand button using the arrow keys, then press Enter to activate it.
+- When scrolling down, the header row remains visible, allowing users to reference column labels while interacting with nested rows.
```

Comment:
The scroll direction doesn't seem relevant, the header row will be displayed no matter what direction the user is scrolling.

```suggestion
- When scrolling, the header row remains visible, allowing users to reference column labels while interacting with nested rows.
```